libraries/azure-app-configuration-importer/src/featureFlag.ts

```diff
@@ -0,0 +1,127 @@
+import { FeatureFlagValue as FeatureFlagValueV1 } from "@azure/app-configuration";
+import { ClientFilter } from "./models";
+
+export interface FeatureFlagValue extends FeatureFlagValueV1 {
+    id: string;
+    description?: string;
+    enabled: boolean;
+    conditions: {
+        clientFilters: ClientFilter[];
+        requirement_type?: RequirementType
+    };
+    displayName?: string;
+    allocation?: Allocation;
+    variants?: Variant[];
+    telemetry?: {
+        enabled?: boolean;
```

Comment:
Telemetry also has a property called, [metadata](https://github.com/microsoft/FeatureManagement/blob/7c4b57e40dcbf7c15a018188311814d821f45c27/Schema/FeatureFlag.v2.0.0.schema.json#L325)

Reply:
Good catch! This has been updated